tests/integration_tests/style/test_rust.py

```diff
@@ -115,8 +133,22 @@ def test_ensure_mod_tests():
 
 def test_for_valid_licenses():
```

Comment:
I think we could move this test to a new file -> `test_license.py`, since it performs the checks for python and bash files, not only rust.

Reply:
That is a very good idea, I did the update, take a look!